doc/source/timeseries.rst

```diff
@@ -21,69 +21,112 @@
 Time Series / Date functionality
 ********************************
 
-pandas has proven very successful as a tool for working with time series data,
-especially in the financial data analysis space. Using the NumPy ``datetime64`` and ``timedelta64`` dtypes,
-we have consolidated a large number of features from other Python libraries like ``scikits.timeseries`` as well as created
+pandas contains extensive capabilities and features for working with time series data for all domains.
+Using the NumPy ``datetime64`` and ``timedelta64`` dtypes, pandas has consolidated a large number of
+features from other Python libraries like ``scikits.timeseries`` as well as created
 a tremendous amount of new functionality for manipulating time series data.
 
-In working with time series data, we will frequently seek to:
+For example, pandas supports:
 
-* generate sequences of fixed-frequency dates and time spans
-* conform or convert time series to a particular frequency
-* compute "relative" dates based on various non-standard time increments
-  (e.g. 5 business days before the last business day of the year), or "roll"
-  dates forward or backward
+Parsing time series information from various sources and formats
 
-pandas provides a relatively compact and self-contained set of tools for
-performing the above tasks.
+.. ipython:: python
+
+   dti = pd.to_datetime(['1/1/2018', np.datetime64('2018-01-01'), datetime(2018, 1, 1)])
+   dti
 
-Create a range of dates:
+Generate sequences of fixed-frequency dates and time spans
 
 .. ipython:: python
 
-   # 72 hours starting with midnight Jan 1st, 2011
-   rng = pd.date_range('1/1/2011', periods=72, freq='H')
-   rng[:5]
+   dti = pd.date_range('2018-01-01', periods=3, freq='H')
+   dti
 
-Index pandas objects with dates:
+Manipulating and converting date times with timezone information
 
 .. ipython:: python
 
-   ts = pd.Series(np.random.randn(len(rng)), index=rng)
-   ts.head()
+   dti = dti.tz_localize('UTC')
+   dti
+   dti.tz_convert('US/Pacific')
 
-Change frequency and fill gaps:
+Resampling or converting a time series to a particular frequency
 
 .. ipython:: python
 
-   # to 45 minute frequency and forward fill
-   converted = ts.asfreq('45Min', method='pad')
-   converted.head()
+   idx = pd.date_range('2018-01-01', periods=72, freq='H')
+   ts = pd.Series(np.random.randn(len(idx)), index=idx)
```

Comment:
I'd prefer `range` over `randn`. The data won't also be deterministic, but when you do the mean, it'll be more clear, and the means will increase.

Also, may be it's difficult, but if you can find a resampling example where the original is much smaller than 72, I think it'd help users if we show the original data before showing the transformed data.